ecosystem/sep-0023.md

```diff
@@ -118,15 +118,15 @@ accounts in a backwards-compatible way:
       account.  (For non-multiplexed accounts, this will be the same
       as `account_id`.)
 
-    - Base field name + `_id` is the integer.
+    - Base field name + `_muxer` is the integer.
```

Comment:
how about `mid`? I'm not really sure what a `muxer` is.

Reply:
A muxer is a thing that does muxing, which highlights that we shouldn't use that term, since the identifier doesn't do the muxing. 🤔 

MID is okay, but in payments an MID is a merchant ID, and it'd be nice not to overload that.

Reply:
How about `_muxed_address` and `_muxed_id`?

e.g.:
In transaction response (unrelated fields omitted):
```json
{
  "source_account": "GAAZI4TCR3TY5OJHCTJC2A4QSY6CJWJH5IAJTGKIN2ER7LBNVKOCCWN7",
  "source_account_sequence": "1",
  "source_account_muxed_address": "M...",
  "source_account_muxed_id": "1",
}
```

Reply:
That looks fine to me.